.rubocop.yml

```diff
@@ -0,0 +1,80 @@
+inherit_from: .rubocop_todo.yml
+
+AllCops:
+  NewCops: enable
+  SuggestExtensions: false
+  TargetRubyVersion: 3.1
+
+#
+# our rules
+#
+
+Layout/FirstArrayElementIndentation: { Exclude: ['spec/**/*'] }
+Layout/LineLength: { Enabled: false }
+Layout/SpaceAroundEqualsInParameterDefault: { EnforcedStyle: no_space }
+Lint/ConstantDefinitionInBlock: { Exclude: ['spec/**/*'] }
+Metrics: { Enabled: false }
+Style/SymbolArray: { EnforcedStyle: brackets }
+Style/IfInsideElse: { Enabled: false } # Offense count: 1
+Style/PercentLiteralDelimiters:
+  Enabled: true
+  PreferredDelimiters:
+    default: '{}'
+    '%i': '[]'
+    '%I': '[]'
+    '%w': '[]'
+    '%W': '[]'
+Style/UnlessElse: { Enabled: false }
+Bundler/OrderedGems: { Enabled: false }
+Style/Next: { Enabled: false }
+Style/HashSyntax: { Enabled: false }
+Naming/BlockForwarding: { Enabled: false }
+Lint/ReturnInVoidContext: { Enabled: false }
+Lint/NoReturnInBeginEndBlocks: { Enabled: false }
+Gemspec/DeprecatedAttributeAssignment: { Enabled: false }
+Layout/EmptyLineAfterMagicComment: { Enabled: false }
+Layout/FirstArgumentIndentation: { Enabled: false }
+Layout/BeginEndAlignment: { Enabled: false }
+Layout/RescueEnsureAlignment: { Enabled: false }
+Style/MultilineBlockChain: { Enabled: false }
+Style/NumericLiterals: { Enabled: false }
+Style/RedundantInterpolation: { Enabled: false }
+Style/FormatString: { Enabled: false }
+Style/RedundantSelf: { Enabled: false }
+Style/FetchEnvVar: { Enabled: false }
+
+#
+# rules that are in flux
+#
+
+# consider enabling these and autocorrecting?
+# Layout/SpaceAfterComma
+# Layout/SpaceAroundKeyword
+# Layout/SpaceBeforeComma
+# Layout/SpaceInsideHashLiteralBraces
+# Layout/SpaceInsideParens
+# Layout/TrailingWhitespace
+# Lint/UnreachableLoop
+# Lint/UnusedBlockArgument
+# Style/ClassCheck
+# Style/Documentation
+# Style/ExpandPathArguments
+# Style/GlobalStdStream
+# Style/HashSyntax
+# Style/KeywordParametersOrder
+# Style/MethodCallWithoutArgsParentheses
+# Style/MutableConstant
+# Style/QuotedSymbols: { EnforcedStyle: double_quotes }
+# Style/RedundantReturn
+# Style/SafeNavigation
+# Style/SpecialGlobalVars
+# Style/StringLiterals: { EnforcedStyle: double_quotes }
+# Style/WordArray
+
```

Comment:
Missing the other disabled rules from the [`.rubocop.yml` template](https://github.com/ronin-rb/ronin-template/blob/b107a4b0e0e6e7836c88f4cd26d1522e90c8d989/.rubocop.yml#L88-L155).